generate/efi/Makefile.config

```diff
@@ -167,7 +168,8 @@ CWARNINGFLAGS = \
 	-Wlogical-op\
 	-Wmissing-parameter-type\
 	-Wold-style-declaration\
-	-Wtype-limits
+	-Wtype-limits\
+	-Wno-error=logical-op
 #
```

Comment:
I didn't see problems by enabling -Wlogical-op.
I guess you have trouble because of different issue.
Could you paste your build error here.